config/integrateddynamics.cfg

```diff
@@ -1,7 +1,7 @@
 # Configuration file
 
 biome {
-    B:biome_meneglin=false
+    B:biome_meneglin=true
```

Comment:
This config option may corrupt BoP worldgen.
Please check this issue and report if its not happens again

https://github.com/EnigmaticaModpacks/Enigmatica2Expert/issues/1895